### DISCUSSION TITLE
Make install - /tmp/distribution.Oyr4Ku" not found

### DISCUSSION THREAD

**Discussion by devoptchous** [goharbor/harbor#18863] [Q&A]
Hello, do somebody had this issue by running make install?

unable to prepare context: path "/tmp/distribution.Oyr4Ku" not found
make[1]: *** [/home/tchous/harbor/make/photon/Makefile:177: _build_registry] Error 1
make[1]: Leaving directory '/home/tchous/harbor'
make: *** [Makefile:381: build] Error 2

Need help. 
THanks,

**Comment by stonezdj** [comment]
What is the context of your command?  does this folder exist?
